### PR TITLE
Cleanup wireguard reference manual

### PR DESCRIPTION
**Pull request by kprosise**
The remote access section of the reference manual (wireguard) was cleaned up to adhere to the style-guide. Added additional content for troubleshooting, and worked on clarifying potential points of confusion.

QA steps: Ran link-check and linter. Checked build output, no issues found with rendering.

This commit addresses FFTK-1704
This commit addresses FFTK-2794
This commit addresses FFTK-2954

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

Feel free to suggest additional info or changes!

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 9c093361ac3332bb8b7da9875c4551327a288a24 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2526/docs/artifacts/html/index.html

**Review by mwasilew — APPROVED** [review]

File: source/reference-manual/remote-access/wireguard.rst [hunk 1]
```diff
@@ -23,30 +21,33 @@ Actions on VPN Server
...
 
-   $ git clone https://github.com/foundriesio/factory-wireguard-server/
-   $ cd factory-wireguard-server
+.. prompt:: bash
 
+     git clone https://github.com/foundriesio/factory-wireguard-server/
+     cd factory-wireguard-server
 
-Configure the Factory with this new service and enable the daemon::
+Configure you Factory with this new service, and enable the daemon:
```
Comment (doanac) [inline source/reference-manual/remote-access/wireguard.rst:39]:
```suggestion
Configure your Factory with this new service, and enable the daemon:
```

**Comment by doanac** [comment]
Docs for ea1f47a6b774019f9720cd6fafe815fbb681739e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2528/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Comment by doanac** [comment]
Docs for b82a5c94f9ea7adf05ff621db25bc2c747d5ea02 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2529/docs/artifacts/html/index.html

**Pull request merged**